tools/codegen/cmd/featureset-markdown.go

```diff
@@ -157,6 +157,9 @@ func getOrderedFeatureGates(info map[string]map[string]*featureGateInfo) []strin
 			for _, featureGate := range byFeature.enabled.List() {
 				counts[featureGate] = counts[featureGate] + 1
 			}
+			for _, featureGate := range byFeature.disabled.List() {
+				counts[featureGate] = counts[featureGate] + 0
```

Comment:
Ineffective assignment? Did you mean add 0?

Reply:
> Ineffective assignment? Did you mean add 0?

I think so. Just making it clear how to bump in the list and nice and parallel. Similar blocks exist in other repos that do shift priority.